pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -137,7 +142,9 @@ let
 
     inherit doCheck;
 
-    meta = apparmor-meta "user-land utilities";
+    meta = apparmor-meta "user-land utilities" // {
+      broken = !(withPython && withPerl);
```

Comment:
Why is this broken?

Reply:
It requires those features to work.

Reply:
Sorry thought this was part of the same derivation as above.